apps/authentication-service/cypress/e2e/clients.cy.ts

```diff
@@ -0,0 +1,74 @@
+import { Client } from "auth0";
+
+describe("Client API", () => {
+  const PORT: number = (Cypress.env("PORT") as number) || 8089;
+  let accessToken: string;
+  let newClientId: string;
+
+  const getToken = () => {
+    cy.window()
+      .its("localStorage")
+      .invoke("getItem", "jwt")
+      .then((token) => {
+        if (token) {
+          accessToken = token;
+        }
+      });
+  };
+
+  before(() => {
+    getToken();
+    if (!accessToken) {
+      cy.loginByAuth0Api();
+      cy.wait(1500);
+      getToken();
+    }
+  });
+
+  it("creates new client", () => {
+    const clientName = "__Test Client__";
+
+    cy.request({
+      method: "POST",
+      url: `http://localhost:${PORT}/v1/clients`,
+      headers: {
+        ContentType: "application/json",
+        Authorization: `Bearer ${accessToken}`,
+      },
+      body: {
```

Comment:
I don't think we want this api to accept a name and a description. Shouldn't these be generated by our api?

Reply:
This has been updated so the api generates an id.